### PR TITLE
Add statistics for lua in-VM runtime and GC operations.

### PR DESCRIPTION
**Pull request by postwait**

### CODE REVIEW

File: src/modules/lua.c [hunk 1]
```diff
@@ -57,6 +59,8 @@ static mtev_hash_table mtev_lua_states;
 static pthread_mutex_t mtev_lua_states_lock = PTHREAD_MUTEX_INITIALIZER;
 static mtev_hash_table mtev_coros;
 static pthread_mutex_t coro_lock = PTHREAD_MUTEX_INITIALIZER;
+static stats_ns_t *lua_stats_ns;
+static stats_handle_t *gc_total, *gc_full, *gc_latency;
```
Comment (cody271) [inline src/modules/lua.c]: We could also report `LUA_GCCOUNT` here fairly easily.

File: src/modules/lua_general.c [hunk 2]
```diff
@@ -810,7 +839,9 @@ mtev_console_lua_repl_execute(mtev_console_closure_t ncct,
   lua_remove(L, -2);  /* pop extras */
   lua_pushstring(L, buff);
   lua_pushConsole(L, ncct);
-  rv = lua_pcall(L, 2, LUA_MULTRET, -4);
+  VM_TIME_BEGIN
+  rv = lua_pcall(L, 3, LUA_MULTRET, -4);
```
Comment (cody271) [inline src/modules/lua_general.c]: Why are we changing `2` to `3` here ??

Comment (postwait) [reply src/modules/lua_general.c]: OUCH. Editor accident

File: src/modules/lua.c (hunk 1 above)
Comment (postwait) [reply src/modules/lua.c]: it's a bit challenging to do it across all the VMs

**Pull request merged**